Sources/CLI/Application.swift

```diff
@@ -112,42 +109,6 @@ struct Application: AsyncParsableCommand {
         return PluginLoader(pluginDirectories: pluginDirectories, pluginFactories: pluginFactories, defaultResourcePath: statePath, log: log)
     }()
 
-    func validate() throws {
```

Comment:
NOTE: no changes...just reordered private functions to the end of the compilation unit